### PR TITLE
th-96: Add custom map design and tow-truck icon

### PR DESCRIPTION
**Pull request by h0wter**
Created a custom design for the map and added a tow-truck icon.

map

### CODE REVIEW

File: frontend/src/libs/packages/map/map.package.ts [hunk 1]
```diff
@@ -30,6 +32,7 @@ class MapService implements IMapService {
     this.map = new google.maps.Map(mapElement, {
       center,
       zoom,
+      mapId: '2db763871084f8b6',
```
Comment (albsbz) [inline frontend/src/libs/packages/map/map.package.ts]: If this from google project, maybe better move it to .ENV?

Comment (Druxight) [inline frontend/src/libs/packages/map/map.package.ts]: use JSON config

**Review by Druxight — CHANGES_REQUESTED** [review]

File: frontend/src/libs/packages/map/map.package.ts [hunk 2]
```diff
@@ -1,6 +1,9 @@
+import truckImg from 'src/assets/img/tow-truck.png';
```
Comment (PavloSukhinin) [inline frontend/src/libs/packages/map/map.package.ts]:
```suggestion
import truckImg from '~/assets/img/tow-truck.png';
```

File: frontend/src/libs/packages/map/map.config.json [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+      {
+        "visibility": "off"
+      }
+    ]
+  },
+  {
+    "featureType": "poi.sports_complex",
+    "stylers": [
+      {
+        "visibility": "off"
```
Comment (kalinkaaaa14) [inline frontend/src/libs/packages/map/map.config.json:99]: quick question - do we need to add such many feature types with visibility off? are they enabled by default?

Comment (h0wter) [reply frontend/src/libs/packages/map/map.config.json:99]: The standard map shows all places, captions. To match the design I had to turn off various points of interest. I turned off only what does not match the design.

Comment (kalinkaaaa14) [reply frontend/src/libs/packages/map/map.config.json:99]: ok, got it, thank you

**Review by kalinkaaaa14 — APPROVED** [review]

**Review by DmytroVoronkov — APPROVED** [review]

**Review by Druxight — APPROVED** [review]

**Pull request merged**